src/index.ts

```diff
@@ -246,6 +247,38 @@ class NotificationAPIClient implements NotificationAPIClientInterface {
       });
     }
   }
+
+  identify = async (user: UserParams): Promise<void> => {
```

Comment:
You can use this line to make it more readable. 
`const { clientId, userId, userIdHash } = this.state.initOptions;`